lib/linguist/heuristics.yml

```diff
@@ -169,6 +182,15 @@ disambiguations:
     pattern: '^\s*(?:%%|main\s*\(.*?\)\s*->)'
   - language: JavaScript
     pattern: '(?m:\/\/|("|'')use strict\1|export\s+default\s|\/\*.*?\*\/)'
+- extensions: ['.ex']
+  rules:
+  - language: Elixir
+    pattern:
+    - '^\s*@moduledoc\s+'
+    - '^\s*(cond|import|quote|unless)\s+'
```

Comment:
To improve performance, it's best to use non-capturing groups `(?:foo|bar)` instead of capturing groups `(foo|bar)` unless backreferences are needed:

~~~suggestion
    - '^\s*(?:cond|import|quote|unless)\s'
~~~